src/Sylius/Behat/Page/Admin/Promotion/UpdatePageInterface.php

```diff
@@ -19,6 +19,17 @@
  */
 interface UpdatePageInterface extends BaseUpdatePageInterface
 {
+
+    /**
+     * @param int $priority
+     */
+    public function setItsPriority($priority = null);
```

Comment:
I'd rather not use default values here, being explicit looks a lot better in this case.

`$page->setItsPriority()` vs `$page->setItsPriority(null)`